ext/ffi_c/extconf.rb

```diff
@@ -8,7 +8,11 @@ def system_libffi_usable?
     # We need pkg_config or ffi.h
     libffi_ok = pkg_config("libffi") ||
         have_header("ffi.h") ||
-        find_header("ffi.h", "/usr/local/include", "/usr/include/ffi")
+        find_header("ffi.h",
+                    "/usr/local/include",
+                    "/usr/include/ffi",
+                    "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/ffi",
```

Comment:
I suggest to simplify and just remove this path since it doesn't seem to have libffi.

Reply:
Correction: that path has it too but seems less reliable and more likely to move in newer macOS versions.